Other files to distribute/MLLEProfile - AGA.ini

```diff
@@ -197,4 +204,145 @@ BoolText=true
 137=Dot                  |-|SignPost|Dot    |Arrow
 138=Snowman Door         |-|Ice     |Snowm  |Door   |???:AL|???:AL|???:AL|???:AL|???:AL|???:AL|???:AH
 139=Yakko Start          |-|StartPos|Yakko  |Start  |???:AL|???:AL
-140=DownLeft Arrow       |-|Signpost|DLeft  |Arrow
\ No newline at end of file
+140=DownLeft Arrow       |-|Signpost|DLeft  |Arrow
+
+141=Barrel A             |-|Franks  |Barrel |#1     |???:AL|???:AL|???:AL|???:AL|???:AL|???:AL|???:AL|???:AL|???:AL|???:AL|???:AL|???:AL|???:AL|???:AL|???:AH
```

Comment:
Do all these remaining events really have exactly the same (very long) list of parameters? This seems improbable

Reply:
They don't, I got the events by editing the bres files in Notepad++ and was unable to figure out the parameter list per event that way. I tried adding this long list for every event and it seemed to work so I chose to keep it like this rather than not have any parameters.